c2corg_ui/templates/outing/edit.html

```diff
@@ -285,7 +285,7 @@ <h2 class="heading show-phone"><span><span translate>weather</span> & <span tran
 
             ## CONDITION RATING
             <div class="form-group half" ng-class="{'has-success': outing.condition_rating}" ng-init="condition_ratings = ${condition_ratings}">
-              <label translate>condition_rating</label>
+              <label app-context-help context-help-title="{{'condition_rating' | translate}}" context-help-content="{{'Global judgment on the conditions encountered on the route during the outing. This criterion can be used to filter outings. The <i>Conditions</i> field allows to fill in additional information.' | translate}}" translate>condition_rating</label>
               <div class="input-group">
```

Comment:
For some reason the "info" icon is not shown here (ditto for "frequentation" and "associatied participants") whereas it's OK for the other attributes with a contextual help: 
![capture du 2017-08-23 16-31-24](https://user-images.githubusercontent.com/1192331/29621300-a1e0f640-8820-11e7-8bc8-347a92f1a6b3.png)

Perhaps you have forgotten something?